tests/test_pyghooks.py

```diff
@@ -232,6 +238,9 @@ def colorizable_files():
                     )
                 elif k == "ow":
                     os.chmod(file_path, stat.S_IWOTH | stat.S_IRUSR | stat.S_IWUSR)
+                elif k == "mh":
+                    os.rename(file_path, file_path + "_target")
+                    os.link(file_path + "_target", file_path)
```

Comment:
Hard links are all equivalent, so this could be simplified to e.g.
```suggestion-2+1
                    os.link(file_path, file_path + "_alias")
```